llvm/lib/Transforms/IPO/FunctionImport.cpp

```diff
@@ -588,7 +568,11 @@ class WorkloadImportsManager : public ModuleImportsManager {
     std::map<std::string, std::vector<std::string>> WorkloadDefs;
     json::Path::Root NullRoot;
     // The JSON is supposed to contain a dictionary matching the type of
-    // WorkloadDefs.
+    // WorkloadDefs. For example:
```

Comment:
Suggest moving this example to the option definition.

Reply:
Done